hera_pspec/pspecdata.py

```diff
@@ -1640,79 +1642,282 @@ def Jy_to_mK(self, beam=None):
-    single list of baselines into the input needed by PSpecData.pspec(bls1, bls2, ...).
+    Create a PSpecData object, run OQE delay spectrum estimation and write 
+    results to a PSpecContainer object.
 
     Parameters
     ----------
-    bls : list of baseline tuples, Ex. [(1, 2), (2, 3), (3, 4)]
+    dsets : list
+        Contains UVData objects or string filepaths to miriad files
+
+    filename : str
+        Output filepath for HDF5 PSpecContainer object
+
+    groupname : str
+        Groupname of the subdirectory in the HDF5 container to store the 
+        UVPSpec objects in. Default is a concatenation the dset_labels.
+
+    dset_labels : list
+        List of strings to label the input datasets. These labels form
+        the psname of each UVPSpec object.
+
+    dset_pairs : list of len-2 integer tuples
+        List of tuples specifying the dset pairs to use in OQE estimation.
+        Default is to form all N_choose_2 pairs from input dsets.
+
+    spw_ranges : list of len-2 integer tuples
+        List of tuples specifying the spectral window range. See
+        PSpecData.pspec() for details. Default is the entire band.
+
+    pol_pairs : list of len-2 tuples
+        List of string or integer tuples specifying the polarization
+        pairs to use in OQE with each dataset pair in dset_pairs.
+        Default is to get all unique pols in the datasets and to form
+        all auto-pol pairs. See PSpecData.pspec() for details.
+
+    blpairs : list of tuples
+        List of tuples specifying the desired baseline pairs to use in OQE.
+        Ex. [((1, 2), (3, 4)), ((1, 2), (5, 6)), ...]
+        The first bl in a tuple is drawn from zeroth index of a tuple in 
+        dset_pairs, while the second bl is drawn from the first index.
+        See pspecdata.construct_blpairs for details. If None, the default
+        behavior is to use the antenna positions in each UVData object to 
+        construct lists of redundant baseline groups to to take all
+        cross-multiplies in each redundant baseline group.
+
+    input_data_weight : string
+        Data weighting to use in OQE. See PSpecData.pspec for details.
+
+    norm : string
+        Normalization scheme to use in OQE. See PSpecData.pspec for details.
+
+    taper : string
+        Tapering to apply to data in OQE. See PSpecData.pspec for details.
+
+    exclude_auto_bls : boolean
+        If blpairs is None, redundant baseline groups will be formed and
+        all cross-multiplies will be constructed. In doing so, if
+        exclude_auto_bls is True, eliminate all instances of a bl crossed 
+        with itself.
+
+    exclude_permutations : boolean
+        If blpairs is None, redundant baseline groups will be formed and
+        all cross-multiplies will be constructed. In doing so, if
+        exclude_permutations is True, eliminates instances of
+        (bl_B, bl_A) if (bl_A, bl_B) also exists.
+
+    group : boolean
```

Comment:
Do we need this argument? We can just assume that `group=True` if `Nbls_per_group` is not None, can't we?

Reply:
true